### PR TITLE
fix: adjust quotes support GITL-164

### PR DESCRIPTION
**Pull request by ealanisg**

### CODE REVIEW

File: src/resources/action-map/filter.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ module.exports = {
   }),
   search: (l, r) => ({
     bool: {
-      filter: (r.match(/(?:[a-zA-Z0-9À-ÖØ-öø-ÿ]|\b[-'’]\b)+/g) || [])
+      filter: (r.match(/(?:[a-zA-Z0-9À-ÖØ-öø-ÿ'’"“”]|\b-\b)+/g) || [])
```
Comment (ealanisg) [inline src/resources/action-map/filter.js:114]: This was only allowing the quotes in the middle of string, not "quoted" by them.
In the example provided they were using very rare quotes, I added support for them too...  `“”`

https://regex101.com/r/0DNDcU/1

File: test/resources/action-map/filter.spec.js [hunk 2]
```diff
@@ -25,6 +25,19 @@ describe('Testing search filter', () => {
...
+  it('Testing string containing quotes', () => {
+    expect(normalize('Use this "offer" so it’s permanently “Unavailable”'))
+      .to.deep.equal([
+        'Use*',
+        'this*',
+        '"offer"*',
+        'so*',
+        'it’s*',
+        'permanently*',
+        '“Unavailable”*'
```
Comment (simlu) [inline test/resources/action-map/filter.spec.js:37]:
```suggestion
        'Unavailable*'
```

File: test/resources/action-map/filter.spec.js [hunk 3]
```diff
@@ -25,6 +25,19 @@ describe('Testing search filter', () => {
       .to.deep.equal([]);
   });
 
+  it('Testing string containing quotes', () => {
+    expect(normalize('Use this "offer" so it’s permanently “Unavailable”'))
+      .to.deep.equal([
+        'Use*',
+        'this*',
+        '"offer"*',
```
Comment (simlu) [inline test/resources/action-map/filter.spec.js:33]:
```suggestion
        'offer*',
```

File: src/resources/action-map/filter.js (hunk 1 above)
Comment (simlu) [reply src/resources/action-map/filter.js:114]: I don't think we want this.

Comment (simlu) [reply src/resources/action-map/filter.js:114]: The test is valuable to keep though, so I'm not going to close this pr (yet)

Comment (ealanisg) [reply src/resources/action-map/filter.js]: Rolled back the change in the regex, only keeping the test.
No changes made on service-search rewriter, seems to be working right.

**Review by simlu — APPROVED** [review]

**Pull request merged**